app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,12 @@
   </head>
 
   <body>
+    <% flash.each do |type, msg| %>
+      <div>
+        <%= msg %>
+      </div>
+    <% end %>
+
```

Comment:
I've added a "Little Etsy Shoppe" here, too(on my branch).  I'll move it to line 18.